storage/responses/google-classroom/google-classroom-submission.json

```diff
@@ -0,0 +1,9 @@
+{
+    "submission": {
+        "id": "Cg4I4uew5KIEEKa_h6_oBQ",
+        "course_id": "199817397960",
+        "coursework_id": "199814668198",
```

Comment:
Same question for these 3 ids.

Reply:
Here the 'id' is a submission id, which is a string.
While 'course_id' and 'coursework_id' although are numeric identifiers in the response, but in the api endpoint, they are a string.
https://developers.google.com/classroom/reference/rest/v1/courses.courseWork.studentSubmissions/get

So, therefore I specified them as a string in form request validation.

I am open to suggestions.